src/services/experimental-features.js

```diff
@@ -0,0 +1,23 @@
+export class ExperimentalFeatures {
+  constructor(storage) {
+    this.storage = storage;
+    this.features = this.storage.getItem('dopplerExpermiental')
+      ? JSON.parse(this.storage.getItem('dopplerExpermiental'))
+      : null;
+  }
+  getExperimentalFeatures() {
+    return this.features;
+  }
+  isFeatureEnabled(feature) {
+    const experimentalFeatures = this.getExperimentalFeatures();
+    return experimentalFeatures && !!experimentalFeatures[feature];
+  }
+  getFeatureValue(feature, param) {
```

Comment:
Why not simply `getFeature(featureName)`, then you can do whatever you want with the result. You do not need `isFeatureEnabled(featureName)` neither.

Reply:
I did it this way just on case one day we want to change the structure of features, that is why validating if it exists is one thing, and getting a value apart, I don't want to have to know how it is implemented in every place I use this, right?

Reply:
In my opinion, each _experimental feature_ could have his own _experimental data_ without restriction about structure, it could be `true`, `5`, `"Any string"`, `{ testProperty: 1, another: "This is a string" }`. We do not need to pay attention to the payload format.

Regarding change structure of the features, it is not so much important if we do not define it. The experimental features will be dying and we can use an arbitrary structure for the new ones.

Reply:
A sound argument, thanks!